.github/files/setup-wordpress-env.sh

```diff
@@ -128,6 +128,43 @@ for PLUGIN in projects/plugins/*/composer.json; do
 	echo "::endgroup::"
 done
 
+# Install WooCommerce plugin used for some Jetpack integration tests.
+#DEFAULT_PHP_VERSION=$(bash -c 'source .github/versions.sh; echo $PHP_VERSION')
+if [[ "$WITH_WOOCOMMERCE" == true ]] &&
+	jq --argjson changed "$CHANGED" -ne '$changed["plugins/jetpack"] // false' > /dev/null
```

Comment:
Not sure about this line, leftover from previous commit. The goal is to only consider Jetpack changed and nothing else when `$WITH_WOOCOMMERCE` is `true`